app/assets/javascripts/discourse/controllers/composer.js.es6

```diff
@@ -109,6 +109,25 @@ export default Ember.Controller.extend({
             !creatingPrivateMessage;
   },
 
+  @computed('model.whisper', 'model.unlistTopic')
+  whisperOrUnlistTopic(whisper, unlistTopic) {
```

Comment:
This can also be written as `whisperOrUnlistTopic: Ember.computed.or("model.whisper", "model.unlistTopic")`